src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -62,11 +62,13 @@ object DepositHandler {
 
   def finalizeDeposit(mimeType: String)(implicit id: String): Try[Unit] = {
```

Comment:
**general remark on the `implicit id: String` parameters in many of these methods:** are you sure this `String` is required everywhere you put it? Can't you remove it in some instances? Not sure, but it seems a bit much...